app/validators/unfinished_training_participant_profile_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UnfinishedTrainingParticipantProfileValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    record.errors.add(attribute, I18n.t("errors.participant_profile.not_a_participant_profile")) unless participant_profile?(value)
+    record.errors.add(attribute, I18n.t("errors.participant_profile.training_complete")) if completed_training?(value)
+  end
+
+  def participant_profile?(instance)
+    instance.is_a?(ParticipantProfile)
+  end
+
+  def completed_training?(instance)
+    instance&.completed_training?
```

Comment:
maybe this is something we can hook into @ethax-ross's work?

Reply:
This is the relevant method FYI https://github.com/DFE-Digital/early-careers-framework/blob/main/app/models/participant_profile/ect.rb#L25C12-L25C59